lib/commands/config.js

```diff
@@ -266,6 +266,9 @@ ${defData}
     if (!long) {
       msg.push(
         `; node bin location = ${process.execPath}`,
+        `; node version = ${process.version}`,
+        `; npm bin location = ${dirname(dirname(__dirname))}`,
```

Comment:
Is `this.npm.bin` incorre

Reply:
I think it's the same in this case, but you're right that its a better fit. I grabbed this from npm.usage but I think we do that there because config isnt loaded yet.